CSC322-Grinnell/mayflower-directory
# Everything for the Department page (except for staff)

Resolves #179  and #180 

# Changes
Added a department tab that lists all the departments. Each department links to a dynamic page that admins could modify. A department has many services, and admins may edit, add, or delete service for a department in the edit page.

# Testing

1. Run `git checkout LynBranch`
2. Run `rails db:migrate`
3. Run `bundle install`
4. Run `rails test`, and make sure all test cases pass.
5. Start the server, and click on the department tab and create an arbitrary department.
6. Click on the department you just created and try to edit it. Make sure you (1) add some services, and (2) edit/ delete some service.

Important Note (credits to Dennis): 
- make sure you re-serve the server after cocoon install
- run `gem uninstall bundler --version 2.0.1` if you get a conflict message on when running `bundle ...`
